src/ConjugatePriors.jl

```diff
@@ -4,10 +4,19 @@ module ConjugatePriors
 
 using PDMats
 using Distributions
+using StatsFuns
 
 import Base: mean
 import Base.LinAlg: Cholesky
 
+import PDMats:
+    PDMat
+
+import StatsFuns:
+    isApproxSymmmetric,
```

Comment:
this is in `Distributions` (https://github.com/JuliaStats/Distributions.jl/blob/c13f0b6748b6f6a5353f47f23133d2cdd838820a/src/utils.jl#L118), as is `hasCholesky`, so can you take those two functions from there instead?

```
using Distributions:
    isApproxSymmmetric,
    hasCholesky
```